src/app/builders/components/Table/BuilderDataRow.tsx

```diff
@@ -1,364 +1,31 @@
 'use client'
```

Comment:
Im not sure if we need this component. 

Maybe we can include it in the `BuildersTable` as we have it in the `BackersRewardTable`